src/Modules/OneTapLogin.php

```diff
@@ -86,17 +86,17 @@ public function name(): string {
 	 */
 	public function init(): void {
 		if ( $this->settings->one_tap_login ) {
-			add_action( 'login_enqueue_scripts', [ $this, 'one_tap_scripts' ] );
-			add_action( 'login_footer', [ $this, 'one_tap_prompt' ] );
-			add_action( 'wp_ajax_nopriv_validate_id_token', [ $this, 'validate_token' ] );
-			add_action( 'rtcamp.id_token_verified', [ $this, 'authenticate' ] );
+			add_action( 'login_enqueue_scripts', array( $this, 'one_tap_scripts' ) );
+			add_action( 'login_footer', array( $this, 'one_tap_prompt' ) );
+			add_action( 'wp_ajax_nopriv_validate_id_token', array( $this, 'validate_token' ) );
+			add_action( 'rtcamp.id_token_verified', array( $this, 'authenticate' ) );
```

Comment:
Remove these unnecessary short array syntax changes.

Reply:
Made the requested changes https://github.com/rtCamp/login-with-google/pull/182/commits/d7dd905f21ac22094ac29bd7b1a9f14e7314a456